Purchases/Public/RCPurchaserInfo.m

```diff
@@ -80,12 +82,15 @@ - (void)initializeMetadataWithSubscriberData:(NSDictionary *)subscriberData {
 
 - (void)initializePurchasesAndEntitlementsWithSubscriberData:(NSDictionary *)subscriberData
                                                subscriptions:(NSDictionary *)subscriptions {
-    NSDictionary<NSString *, NSArray *> *nonSubscriptions = subscriberData[@"non_subscriptions"];
-    self.nonConsumablePurchases = [NSSet setWithArray:[nonSubscriptions allKeys]];
+    NSDictionary<NSString *, NSArray *> *nonSubscriptionsData = subscriberData[@"non_subscriptions"];
+    self.nonConsumablePurchases = [NSSet setWithArray:[nonSubscriptionsData allKeys]];
```

Comment:
so `non_subscriptions` filters out consumables as well?

Reply:
Yes, `non_subscriptions` contains both consumables and non-consumables. We've named this  (`nonConsumablePurchases`) wrong since the beginning

Reply:
ohh, so `nonConsumablePurchases` _should_ have been named `nonSubscriptions`. Maybe we should add a warning? Seems like a gotcha

Reply:
in that case, disregard my comment about the sample purchases naming, since consumables would also work